manage_breast_screening/clinics/models.py

```diff
@@ -109,3 +109,31 @@ class ClinicSlot(BaseModel):
     duration_in_minutes = models.IntegerField()
 
     # TODO: add unique constrants
+
+
+class ScreeningEpisode(BaseModel):
+    participant = models.ForeignKey("participants.Participant", on_delete=models.CASCADE)
+
+
+class Appointment(BaseModel):
+    class Status:
+        CONFIRMED = "CONFIRMED"
+        CANCELLED = "CANCELLED"
+        DID_NOT_ATTEND = "DID_NOT_ATTEND"
+        CHECKED_IN = "CHECKED_IN"
+        SCREENED = "SCREENED"
+        ATTENDED_NOT_SCREEN = "ATTENDED_NOT_SCREEN"
```

Comment:
minor naming thing, but should this be `ATTENDED_NOT_SCREENED` for consistency so everything is past tense?

Reply:
absolutely, typo